app/models/poll.rb

```diff
@@ -0,0 +1,9 @@
+class Poll < ActiveRecord::Base
+  has_many :ballots
+  has_many :users, through: :ballots
+
+  def self.current
+    return last unless !any? || last.closed
+      nil
```

Comment:
`last.closed?` The question mark is a ruby-ism to indicate the method will return true or false